hotelReservation/docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       - JAEGER_SAMPLE_RATIO
       - LOG_LEVEL
     build: .
-    image: hotel_reserv_frontend_single_node
+    image: nevenass/dsbnew
```

Comment:
Shouldn't this image be reverted back to the original before merging?

Reply:
Fixed